test_util/azure_test_driver.py

```diff
@@ -123,70 +131,77 @@ def poll_deploy():
         assert deploy_poller.done(), "Deployment failed / polling didn't reach deployment done."
         deployment_result = deploy_poller.result()
         print(deployment_result.properties.outputs)
-        master_lb = deployment_result.properties.outputs['dnsAddress']['value']
-        master_url = "http://{}".format(master_lb)
+        master_lb = deployment_result.properties.outputs['masterFQDN']['value']
 
         print("Template deployed using SSH private key: https://mesosphere.onelogin.com/notes/18444")
-        print("For troubleshooting, master0 can be reached using: ssh -p 2200 core@{}".format(master_lb))
-
-        @retry(wait_fixed=(5 * 1000), stop_max_delay=(15 * 60 * 1000))
-        def poll_on_dcos_ui_up():
-            r = get_dcos_ui(master_url)
-            assert r is not None and r.status_code == requests.codes.ok, \
-                "Unable to reach DC/OS UI: {}".format(master_url)
-
-        print("Waiting for DC/OS UI at: {} ...".format(master_url))
-        poll_on_dcos_ui_up()
+        print("For troubleshooting, master0 can be reached using: ssh -p 2200 {}@{}".format(
+            get_value('linuxAdminUsername'), master_lb))
 
         # Run test now, so grab IPs
         nmc = NetworkManagementClient(credentials, subscription_id)
         ip_buckets = {
-            'masterNodeNic': [],
-            'slavePrivateNic': [],
-            'slavePublicNic': []}
-
-        for resource in rmc.resource_groups.list_resources(group_name):
-            for bucket_name, bucket in ip_buckets.items():
-                if resource.name.startswith(bucket_name):
-                    nic = nmc.network_interfaces.get(group_name, resource.name)
-                    all_ips = []
-                    for config in nic.ip_configurations:
-                        all_ips.append(config.private_ip_address)
-                    bucket.extend(all_ips)
-
-        with closing(SSHTunnel('core', 'ssh_key', master_lb, port=2200)) as t:
+            'master': [],
+            'private': [],
+            'public': []}
+
+        for resource in rmc.resource_groups.list_resources(
+                group_name, filter=("resourceType eq 'Microsoft.Network/networkInterfaces' or "
```

Comment:
nit: Be nice to only check the VMSS for ACS, and only the networkInterfaces for the DCOS templates, but this is reasonable for now.